File: day five/java/CardValidator.java
import java.util.Scanner;
public class CardValidator{

	public static void main(String[] args){

	Scanner scanner = new Scanner(System.in);

	System.out.print("Enter A Card Number: ");

	String cardNumber = scanner.nextLine();

	System.out.print("Card Number Is Valid");


	}

	public static boolean isValid(long number){

	return (getSizenumber)

	getSize(number) == 16) && (prefixMatched(number, 4) || prefixMatched(number, 5) || prefixMatched(number, 6));
			
	}

	
	
	
	}












}

File: day five/python/cardvalidator.py

card_number = str(input("Enter A Card Number: "))

def isValid(card_number):

	return (CreditCard.getSize(card_number) == 16 ) and (CreditCard.prefixMatched(card_number, 4) or CreditCard.prefixMatched(card_number, 5) or CreditCard.prefixMatched(number, 6))